libsrc/gamate/vga.inc

```diff
@@ -0,0 +1,228 @@
+
+; VGA charset for the PC-Engine conio implementation
+
+        .byte   $00, $00, $00, $00, $00, $00, $00, $00
+        ; 1
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11111111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 2
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 3
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11111111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 4
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00011111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 5
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11110000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 6
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00011111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 7
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11110000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 8
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11111111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 9
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11111111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 10 (LF)
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 11
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11110000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 12
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00011111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+
+        .byte   $3F, $33, $3F, $30, $30, $70, $F0, $E0
```

Comment:
Sorry for no asking this for the PC-Engine in the first place, but wouldn't it be nice to have all chars defined using binary constants (%) like done above? Maybe it would be actually worth the effort to hack a simple converter and use it to process the rest of this file - what do you think?

I don't know if there are other scenarios (maybe C64 soft80?) where large piles of constants could be rendered actually comprehensible by using (nicely formatted) binary constants?